tools/quicktype-wrapper/src/index.ts

```diff
@@ -2,26 +2,29 @@
 
 import { readFileSync, writeFileSync } from "fs";
 import {jsonschema2language, LANGUAGE, LANGUAGE_EXT, TARGET_LANGUAGE} from './quickstype';
+const {argv} = require('yargs')
 const mkdirp = require('mkdirp');
 const HOMEDIR = process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'];
 
 /**
  * A simple tool that generates code using quicktype.
+ * 
+ * Configuration (via environment variables or command-line flags):
  * @param {string} IN The directory for JSON Schema input. Must have trailing /.
  * @param {string} OUT The directory for generated output. Must have trailing /.
  * @param {string} L The target language
  */
-const IN = process.env.IN;
-const OUT = process.env.OUT;
-const L = (process.env.L || LANGUAGE.TYPESCRIPT).toUpperCase() as TARGET_LANGUAGE;
+const IN = process.env.IN || argv.in;
```

Comment:
My guess is that this means the environment variable takes precedence over the command line argument - is that the case? If so, I'd suggest reversing it - treat the environment variable as a default, allowing the command line argument to override it.

Reply:
Yeah, should switch this.